src/app/project/project-details-tab/project-details-tab.component.ts

```diff
@@ -44,13 +47,17 @@ export class ProjectDetailsTabComponent implements OnInit, AfterViewInit, OnDest
     this.commentPeriod = this.project.commentPeriodForBanner;
     this.route.data.subscribe((res: any) => {
       if (res) {
-        if (res.shapefile && res.shapefile[0].data.meta && res.shapefile[0].data.meta.length > 0) {
-          this.shapefile = res.shapefile[0].data.searchResults;
+        if (res.documents && res.documents[0].data.meta && res.documents[0].data.meta.length > 0) {
+          this.shapefile = res.documents[0].data.searchResults;
```

Comment:
You might want to consider using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining here, might clean up the code a bit

Reply:
Considered, but not willing to change at this time. Could be code impacts for the empty array that is set below vs whatever happens to the variable when const something?.something else fails.

Reply:
@SteveMHoward Actually, I don't think there's a downside to using optional chaining in this instance. The if statements will still evaluate the same. The difference is:

- Without optional chaining: Any one of those properties missing will cause a nasty error to be thrown saying that there is no such property. We just haven't run into this before. The if statement will _not_ default to the "else" but the code will error out.
- With optional chaining: If any property is missing, "undefined" will be output and the if statement logic will continue as normal.

So, TLDR, optional chaining will actually make it _more_ likely the if statement logic evaluates as expected.